### PR TITLE
developed script

### PR DESCRIPTION
**Pull request by danielsiegl**

### CODE REVIEW

**Review by philippkalenda — APPROVED** [review]
Looks good and works.

File: example-scripts/powershell/publishExisting.ps1 [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#Powershell sees curly brackets as codeblocks so use quotations marks "{}" or just write your guids without them
+$componentGuids =""
+
+$componentFiles = Get-ChildItem *.mpms
```
Comment (FlorianBeerRB) [inline example-scripts/powershell/publishExisting.ps1:4]: Is there a better way to get this information?
E.g. by looking into the EA model and extracting all MPMS definitions with a specific stereotype?
Or maybe by having a json config controlling the export. The MPMS content will have to be changed after each model change and therefore be rarely consistent with the EA model when maintained in GIT. Or we have to add after each push a commit "Updated MPMS" with a workflow

Comment (danielsiegl) [reply example-scripts/powershell/publishExisting.ps1:4]: We need this for a project now - I guess

File: example-scripts/powershell/importWithDependency.ps1 [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿
+$mpmsfile ="Customer A-46e3f67f-4e87-c98e-129a-049eb063405f.mpms"
+$emptyModel = "empty.eapx"
+$newModelName ="Customer A.eapx"
+
+$path = Get-Location
```
Comment (danielsiegl) [inline example-scripts/powershell/importWithDependency.ps1:6]: this is needed - somehow this script behaves wired with the working directory

**Pull request merged**